src/plugin/debugger/mhpdbgc.c

```diff
@@ -2485,7 +2485,6 @@ static int bpchk(unsigned int a1)
 
   for (i1 = 0; i1 < MAXBP; i1++) {
     if (mhpdbgc.brktab[i1].is_valid && mhpdbgc.brktab[i1].brkaddr == a1) {
```

Comment:
The condition probably had to
be changed to
`&& dpmimode == mhpdbgc.brktab[i1].is_dpmi`
to make any sense of this `is_dpmi`
after your removal. But not something
to really care about.